### ISSUE TITLE
Mechanical Shutter supported Key returns false for Mavic 3 Enterprise

### ISSUE THREAD

**Issue by rishabhSkylark** [dji-sdk/Mobile-SDK-Android-V5#174]
Hi, I am trying to build a POC android app for my DJI Mavic 3 Enterprise. I want to GET the key for Mechanical Shutter enabled or not, “KeyIsMechanialShutterSupported”. I am able to get this value from the drone, but the value is false.

When I try to fetch this Key `KeyIsMechanialShutterSupported` on a Mavic 3 Enterprise, I recieved “false” as the response from the drone. Although the drone supports Mechanical Shutter, I can even enable and disable the mechanical shutter from DJI Pilot 2 as well setting the key KeyMechanicalShutterEnable.

This is my code to GET the key KeyIsMechanialShutterSupported. (It is in Kotlin).

```kotlin
KeyManager.getInstance().getValue(
	KeyTools.createKey(CameraKey.KeyIsMechanialShutterSupported),
			object : CommonCallbacks.CompletionCallbackWithParam<Boolean> {
				override fun onSuccess(isSupported: Boolean) {
					println("isMechanicalShutterSupported $isSupported")
				}
				override fun onFailure(error: IDJIError) {
					println("Unable to get value with error: ${error.description()}")
				}
			}
	)
```

I get the `onSuccess` log `isMechanicalShutterSupported false`.

**These are my UAS details**:

**DJI MSDK V5 Version:** 5.6.0
**Aircraft:** DJI Mavic 3 Enterprise
**Aircraft version:** 06.01.0606
**RC:** DJI RC Pro Enterprise (Model: RM510B)
**RC Android Version:** 10

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/92386'>Zendesk ticket #92386</a>:<br><br>I have reproduced the issue you mentioned.
I would like to confirm if there is an interface to determine if the current camera supports a mechanical shutter. Please wait for a moment.
<br><p><em><small>°°°</small></em></p>

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/92386'>Zendesk ticket #92386</a>:<br><br>Sorry for keeping you waiting for so long. The reason for the error with `KeyIsMechanialShutterSupported` is most likely due to a lack of compatibility, but the switch interface for the mechanical shutter can still be used normally.
<br><p><em><small>°°°</small></em></p>

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/92386'>Zendesk ticket #92386</a>:<br><br>The issue has been fixed in version 5.9. You can update after the release to experience the effect.
<br><p><em><small>°°°</small></em></p>
